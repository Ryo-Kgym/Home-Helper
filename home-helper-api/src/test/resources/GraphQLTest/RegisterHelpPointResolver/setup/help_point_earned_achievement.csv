id,user_id,earned_datetime,earned_point
1,u1,2022-12-01,1234