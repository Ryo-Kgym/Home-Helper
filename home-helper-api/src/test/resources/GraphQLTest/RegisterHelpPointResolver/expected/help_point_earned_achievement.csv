id,user_id,earned_date,earned_point
1,uuu,2022-12-01,1234
2,uuu,2022-12-04,2550